v2/client/src/components/pages/survey/Survey.js

```diff
@@ -45,10 +39,28 @@ export default class Survey extends React.Component {
         });
```

Comment:
show separate api calls in actions and make this a presentation component

just hookup to redux to get info

Reply:
https://medium.com/@learnreact/container-components-c0e67432e005

get the fetching functions from props to have better reusability as you can swap in different functions